### DISCUSSION TITLE
请问不同服务器组解析的缓存会 合并 还是 覆盖 还是 互不影响？

### DISCUSSION THREAD

**Discussion by Benjamin1919** [pymumu/smartdns#1606] [Q&A] [answered]
在解析同一个域名的情况下：

【1】假设默认端口（比如6053）的默认组有两个服务器组：office和home，那么这两个服务器组解析的缓存结果会 合并 还是 覆盖 还是 互不影响？

【2】假设6553端口绑定服务器组global，global组排除默认，那么6553端口和6053端口解析的缓存结果会 合并 还是 覆盖 还是 互不影响？

**Comment by PikuZheng** [comment, marked as answer]
>1.2022.v38.1.14
>38.1.10~38.1.14:
>1. 重要！修正了不同服务器组共用缓存问题

所以在38.1.14前是共用缓存，之后是独立缓存
